Adicionada a package de cookies

tiagomgelias opened the pull request.

tiagomgelias committed:
Adicionada a package de cookies

claudio-silva reviewed (changes requested):
Please remove this.
Already merged in.
Thanks

tiagomgelias committed:
Remoção dos métodos xxxField(), mudança de nome do getRecord e setRecord

tiagomgelias committed:
Merge branch 'master' into master

tiagomgelias committed:
Adicionado codigo para o metodo mergeFields

tiagomgelias committed:
Merge remote-tracking branch 'origin/master'

tiagomgelias committed:
-Mudança no funcionamento do metodo mergeFields()
-adicionado o campo 'username' a funcao sleep para poder ser serializado
